CHANGELOG.rst

```diff
@@ -23,7 +23,8 @@ Deprecations:
 Changes:
 ^^^^^^^^
 
-*none*
+- Automatically set `SSL_CTX_set_ecdh_auto()` on ``OpenSSL.SSL.Context``.
```

Comment:
Can you make `SSL_CTX_set_ecdh_auto()` a reST code too pls?  Otherwise LGTM and you can merge it if the tests pass.